apps/lfx-one/src/app/modules/dashboards/org/org-meetings/components/org-upcoming-meetings/org-upcoming-meetings.component.html

```
@@ -119,95 +123,98 @@ <h3 class="text-base font-semibold text-gray-900">{{ meeting.title }}</h3>
-                      <div class="flex items-center gap-3 py-2 first:pt-0 last:pb-0">
-                        <lfx-person-avatar [avatarUrl]="invitee.avatarUrl" [name]="invitee.name" [identity]="invitee.name" />
-                        <div class="flex-1 min-w-0">
-                          <p class="truncate text-sm font-medium text-gray-900">{{ invitee.name }}</p>
-                          <p class="truncate text-sm text-gray-500">{{ invitee.title }}</p>
-                        </div>
-                        <span
-                          class="inline-flex flex-shrink-0 items-center gap-1 rounded-full px-2.5 py-1 text-xs font-medium"
-                          [class]="rsvpBadge(invitee.rsvpStatus).badgeClass">
-                          <i class="fa-light fa-circle-small text-[6px]" aria-hidden="true"></i> {{ rsvpBadge(invitee.rsvpStatus).label }}
-                        </span>
+            @if (meeting.orgInvitees.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)